test/Microsoft.ML.Tests/ScenariosWithDirectInstantiation/TensorflowTests.cs

```diff
@@ -846,5 +848,59 @@ public void TensorFlowTransformCifarInvalidShape()
             }
             Assert.True(thrown);
         }
+
+        /// <summary>
+        /// Class to hold features and predictions.
+        /// </summary>
+        public class TensorFlowSentiment
+        {
+            public string Sentiment_Text;
+            [VectorType(600)]
+            public int[] Features;
+            [VectorType(2)]
+            public float[] Prediction;
+        }
+
+        [ConditionalFact(typeof(Environment), nameof(Environment.Is64BitProcess))]
+        public void TensorFlowSentimentClassificationTest()
+        {
+            var mlContext = new MLContext(seed: 1, conc: 1);
+            var data = new[] { new TensorFlowSentiment() { Sentiment_Text = "this film was just brilliant casting location scenery story direction everyone's really suited the part they played and you could just imagine being there robert  is an amazing actor and now the same being director  father came from the same scottish island as myself so i loved the fact there was a real connection with this film the witty remarks throughout the film were great it was just brilliant so much that i bought the film as soon as it was released for  and would recommend it to everyone to watch and the fly fishing was amazing really cried at the end it was so sad and you know what they say if you cry at a film it must have been good and this definitely was also  to the two little boy's that played the  of norman and paul they were just brilliant children are often left out of the  list i think because the stars that play them all grown up are such a big profile for the whole film but these children are amazing and should be praised for what they have done don't you think the whole story was so lovely because it was true and was someone's life after all that was shared with us all" } };
+            var dataView = mlContext.Data.ReadFromEnumerable(data);
+
+            var lookupMap = mlContext.Data.ReadFromTextFile(@"sentiment_model/imdb_word_index.csv",
+                   columns: new[]
+                   {
+                        new TextLoader.Column("Words", DataKind.TX, 0),
+                        new TextLoader.Column("Ids", DataKind.I4, 1),
+                   },
+                   separatorChar: ','
+               );
+
+            // We cannot resize variable length vector to fixed length vector in ML.Net
+            // The trick here is to create two pipelines.
+            // The first pipeline tokenzies the strings into words and maps the words to an integer which is an index in the dictionary.
+            // Then this integer vector is retrieved from the pipeline and resized to fixed length.
+            // The second pipeline takes the resized integer vector and passed to TensoFlow and get the classification scores.
```

Comment:
It's not straightforward to user to understand what are "first pipeline" and "second pipeline". Please cite their variable names here or move those statements to the associated pipeline declarations.